beta/src/stores/tracks.types.ts

```diff
@@ -0,0 +1,20 @@
+export interface Track {
```

Comment:
Had to put these in a separate file since we can't export multiple things from the component files for now (see other comment).